### DISCUSSION TITLE
Mounting zfs

### DISCUSSION THREAD

**Discussion by bananabrick** [openzfs/zfs#12394] [General]
Hey, I was trying to mount a couple of drives, but the mounting fails, if I mount using the link in `/dev/disk/by-id/....`, but it succeeds if I use the device name directly.

Full steps here: https://gist.github.com/bananabrick/53aff8527157325ee276e85dc47ef6bb

I need to be able to go through the id, and mount the disks.

**Comment by bghira** [comment]
looks like an issue with the google udev rules.

**Reply by bananabrick** [reply]
This same process works perfectly, when I try to mount an ext4 fs `by-id`.

**Reply by bghira** [reply]
that's not a valid comparison
